validator/api/src/main/java/tech/pegasys/teku/validator/api/KeyStoreFilesLocator.java

```diff
@@ -116,7 +119,8 @@ private boolean hasDepositDataFileContents(final File keyFile) {
     }
   }
 
-  void parseDirectory(final File keyDirectory, final File passwordDirectory) {
+  private Map<Path, Path> parseDirectory(final File keyDirectory, final File passwordDirectory) {
+    Map<Path, Path> pathMap = new HashMap<>();
```

Comment:
Would need to pass `pathMap` in as an arg here too and add to it rather than creating a new map.